plugins/woocommerce/tests/legacy/unit-tests/templates/functions.php

```diff
@@ -193,4 +193,49 @@ public function test_hidden_field() {
 		$this->assertEquals( $expected_html, $actual_html );
 
 	}
+
+	/**
+	 * Test: test_radio_not_required_field.
+	 */
+	public function test_radio_not_required_field() {
+		$actual_html = woocommerce_form_field(
+			'test',
+			array(
+				'type'     => 'radio',
+				'id'       => 'test',
+				'required' => false,
+				'options'  => array(
+					'1' => 'Option 1',
+					'2' => 'Option 2',
+				),
+				'return'   => true,
+			),
+			'1'
+		);
+
+		$this->assertNotContains( 'aria-required', $actual_html );
+	}
+
+	/**
+	 * Test: test_radio_required_field.
+	 */
+	public function test_radio_required_field() {
+		$actual_html   = woocommerce_form_field(
+			'test',
+			array(
+				'type'     => 'radio',
+				'id'       => 'test',
```

Comment:
This should make the test pass
```suggestion
				'id'       => 'test_radio',
```

Reply:
Ope, thank you!